src/lib/Tests/Form/DataTransformer/ContentInfoTransformerTest.php

```diff
@@ -61,11 +63,8 @@ public function testTransformWithInvalidInput($value)
 
     /**
      * @dataProvider reverseTransformDataProvider
-     *
-     * @param $value
-     * @param $expected
      */
-    public function testReverseTransform($value, $expected)
+    public function testReverseTransform($value, $expected): void
```

Comment:
missing strict type

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1148/commits/42437d627f79cccd4225b508580a7512e00f0d8d